src/main/java/uk/gov/companieshouse/charges/data/model/Updated.java

```diff
@@ -3,8 +3,13 @@
 import java.time.LocalDateTime;
 import java.util.Objects;
 
+import org.springframework.format.annotation.DateTimeFormat;
+
 public class Updated {
 
+    @DateTimeFormat(
+            iso = DateTimeFormat.ISO.DATE
```

Comment:
This should be iso = DateTimeFormat.ISO.DATE_TIME

Reply:
Sorted now - Thanks